hermes/probe/create_file_test.go

```diff
@@ -0,0 +1,201 @@
+package probe
+
+import (
+	"fmt"
+	"testing"
+	"bytes"
+	"context"
+
+	"cloud.google.com/go/storage"
+	"github.com/googleapis/google-cloud-go-testing/storage/stiface"
+)
+
+func TestFileName(t *testing.T) {
+	testUnits := [6]RandomHermesFile{ 
```

Comment:
Nit: Use slices instead of arrays (that lets you omit the size).

Reply:
changed